LongWarOfTheChosen/Config/XComGameCore.ini

```diff
@@ -871,7 +871,10 @@ VIPER_ROUNDS_APPLICATION_CHANCE=12
 
 +HackRewardNames="BuffEnemy_FullOverride"
 +HackRewardNames="ControlRobot_Mission"
++HackRewardNames="GreaterShutdownRobot"
++HackRewardNames=GreaterShutdownTurret
```

Comment:
Missed this the first time. Is this going to be OK without the quotes?

Reply:
fixed.